jujugui/static/gui/src/app/components/error-boundary/error-boundary.js

```diff
@@ -0,0 +1,53 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const React = require('react');
+const PropTypes = require('prop-types');
+
+class errorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {error: null};
+  }
+
+  componentDidCatch(error) {
+    this.setState({error});
+  }
+
+  render() {
+    const error = this.state.error;
+    if (error) {
+      return (
+        <div className="v1">
+          <div className="row">
+            <div className="col-12">
+              <div className="p-notification--negative">
+                {error.message &&
```

Comment:
This message will be a JS error and won't be very helpful to the user. I think it should go under the friendly message, maybe in a collapsed box.

Reply:
I've move it below and placed it inside a Vanilla `<code>` block.